### PR TITLE
Add endpoint for plotting metadata

### PR DESCRIPTION
**Pull request by r-ash**
This PR will

- Add an endpoint to return metadata to the front end about plotting. Including:
   - Colour and scale to use per country and indicator
   - How to locate & present data from the uploaded data (column containing value to present, optional filtering on another column, display name for data)

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: hintr
 Title: R API for calling naomi district level HIV model
-Version: 0.0.4
+Version: 0.0.5
```
Comment (richfitz) [inline DESCRIPTION]: Please increment the naomi requirement too (you require 0.0.3 now)

File: R/metadata.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
+do_plotting_metadata <- function(country) {
+  metadata <- naomi::get_plotting_metadata(country)
```
Comment (richfitz) [inline R/metadata.R]: Lines 3-7 can be replaced with (I believe):

```
lapply(split(metadata, metadata$data_type), build_data_type_metadata)
```

and a similar pattern will probably help simplify the functions below

File: R/metadata.R [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  out <- lapply(unique(metadata$indicator), function(indicator) {
+    data <- metadata[metadata$indicator == indicator, ]
+    build_indicator_metadata(data)
+  })
+  names(out) <- unique(metadata$indicator)
+  out
+}
+
+build_indicator_metadata <- function(metadata) {
+  if(nrow(metadata) != 1) {
```
Comment (richfitz) [inline R/metadata.R]: missing space after if (sorry)

File: inst/schema/PlottingMetadataResponse.schema.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
```
Comment (richfitz) [inline inst/schema/PlottingMetadataResponse.schema.json]: it looks like `{survey,anc,output}_chloropleth` refer to the same type, and so you could simplify this

File: DESCRIPTION (hunk 1 above)
Comment (r-ash) [reply DESCRIPTION]: Fixed, thanks.

File: R/metadata.R (hunk 3 above)
Comment (r-ash) [reply R/metadata.R]: Good spot

File: R/metadata.R (hunk 2 above)
Comment (r-ash) [reply R/metadata.R]: Thank you! I wanted something like that, so much neater.

File: inst/schema/PlottingMetadataResponse.schema.json (hunk 4 above)
Comment (r-ash) [reply inst/schema/PlottingMetadataResponse.schema.json]: They do, though this is one of the things I expect might change after talking to Jeff. I've simplified it for now though as you suggest.

**Pull request merged**